extensions/amp-date-countdown/1.0/date-countdown-config.js

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
Let's rename this file to just `messages.js`.

Reply:
Sounds good updated!